atomic_reactor/build.py

```diff
@@ -183,6 +185,29 @@ def set_df_path(self, path):
         for image in dfp.parent_images:
             self.parent_images[image] = None
 
+        # validate user has not specified COPY --from=image
+        builders = []
+        for stmt in dfp.structure:
+            if stmt['instruction'] == 'FROM':
+                match = re.match(r'\S+\s+as\s+(\S+)', stmt['value'], re.I)
+                builders.append(match.group(1) if match else str(len(builders)))
+            if stmt['instruction'] == 'COPY':
+                match = re.match(r'--from=(\S+)', stmt['value'], re.I)
```

Comment:
Could there be white spaces before `--from` ? Or will dockerfile-parse trim the value?

Reply:
rather than test and comment about that, probably just best to handle them by using `search` instead of `match`